enhancing/enhancer.spec.js

```diff
@@ -1,2 +1,77 @@
 const enhancer = require('./enhancer.js');
 // test away!
+
+it('item => new item with durability === 100', () => {
+  const received = {
+    name: 'Sword of Regret',
+    enhancement: 12,
+    durability: 28
+  };
+  const expected = {
+    name: 'Sword of Regret',
+    enhancement: 12,
+    durability: 100
+  };
```

Comment:
this pattern of being extra-explicit for each object you set up for your test is very declarative, but you're going to end up repeating yourself and extending your line count quite a bit. once you feel comfortable with the jest matcher/expected syntax, start cutting down by either declaring the object globally and resetting it with a `beforeEach` method, or at the very least only check the properties you actually want to test for. for this first one for instance, instead of matching the complete objects with `toEqual`, just check the `repair(received).durability` and match it `toBe(100` to save yourself some time and be even more explicit not about what you're passing in, but what specific property you're testing for